components/ADempiere/FieldDefinition/FieldSearch/GeneralInfoSearch/button.vue

```diff
@@ -1,7 +1,7 @@
 <!--
  ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
  Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A. www.erpya.com
- Contributor(s): Edwin Betancourt EdwinBetanc0urt@outlook.com https://github.com/EdwinBetanc0urt
+ Contributor(s): Elsio Sanchez elsiosanches@gmail.com https://github.com/elsiosanchez
```

Comment:
Don't overwrite authors

Reply:
I created this component from the beginning and I made a mistake when I copied the license, I needed to change the author
https://github.com/solop-develop/frontend-default-theme/commit/b334ad281cff3a588bc0692f1818caaec26d38b9#diff-4d34dd521cfbe0c345e82b2cde3f7769efee341c485013bffce6349d8198b65bR4